pkg/storaged/stratis-details.jsx

```diff
@@ -703,6 +887,33 @@ export function start_pool(client, uuid, show_devs) {
                         unlock_with_keydesc(key_desc);
                 }));
     }
+
+    if (stopped_props.key_description &&
+        stopped_props.key_description.t == "(bv)" &&
+        stopped_props.key_description.v[0] &&
+        stopped_props.key_description.v[1].t == "(bs)" &&
```

Comment:
This is moved code, but it does look curious. Would these ever have any other data types? Inspecting types feels odd.

Reply:
Yeah, I don't like this code. I was trying to prevent Cockpit from crashing if ever Stratis has some funny stuff in its properties. I think this excessive encoding is the result of some Rust bindings of Option<foo> things, it doesn't really feel like carefully designed API.

Reply:
Maybe I can abstract this away a bit, by pre-processing it in update_indices.

Reply:
Done.